### ISSUE TITLE
[BUG] Can't create new case from the template via API

### ISSUE THREAD

**Issue by barn4k** [dfir-iris/iris-web#482]
**Describe the bug**
I've created a case template and trying to create a case, using that template. According to [the docs](https://docs.dfir-iris.org/_static/iris_api_reference_v2.0.1.html#tag/Manage-case/operation/post-manage-cases-add), template should be an integer, but the [python code](https://github.com/dfir-iris/iris-web/blob/79cc64ffdff056d7505f189c0964b085710f2a76/source/app/blueprints/manage/manage_cases_routes.py#L387) is trying to treat it as a string, so I'm receiving the error `Error creating case - check server logs`. And in the logs I see:
```
2024-05-24 15:23:01 :: ERROR :: manage_cases_routes :: api_add_case :: Traceback (most recent call last):
  File "/iriswebapp/app/blueprints/manage/manage_cases_routes.py", line 364, in api_add_case
```

If I go to that line, I'll see:
```
[ec2-user@ip-10-245-8-140 ~]$ sudo docker container exec a7779869b204 head -n 364 /iriswebapp/app/blueprints/manage/manage_cases_routes.py | tail -1
        if case_template_id and len(case_template_id) > 0:
```

**To Reproduce**
Sample request
```
{
  "method": "POST",
  "uri": "/manage/cases/add",
  "body": {
    "case_soc_id": "SOC_123",
    "case_customer": 4,
    "case_name": "test name",
    "case_description": "some description",
    "classification_id": 12,
    "case_template_id": 2
  }
}
```

**Expected behavior**
Case will be created based on the template

**Desktop (please complete the following information):**
 - OS: Docker
 - IRIS Version 2.3.7

**Comment by koromodako** [comment]
I fully agree with @barn4k, it seems really strange to process some fields as strings while [the documentation](https://docs.dfir-iris.org/_static/iris_api_reference_v2.0.2.html#tag/Manage-Cases/operation/post-manage-cases-add) explicitly states that this field type is integer or number and JSON allows integer and number as values.
